src/python/system/shell.py

```diff
@@ -369,12 +369,6 @@ def get_interpreter(file_to_execute, is_blackbox_fuzzer=False):
   except KeyError:
     return None
 
-  # TODO(mbarbella): Remove this when fuzzers have been migrated to Python 3.
```

Comment:
Need to keep this if. this is currently used, and then above pylint can be removed.